app/build.gradle

```diff
@@ -54,4 +54,6 @@ dependencies {
     // Required only if Facebook login support is required
     // Find the latest Facebook SDK releases here: https://github.com/facebook/facebook-android-sdk/blob/master/CHANGELOG.md
     // implementation 'com.facebook.android:facebook-login:8.1.0'
+    // Use material design dependency for navbar
+    implementation 'com.google.android.material:material:1.3.0-alpha03'
```

Comment:
Remove since we already import material v1.8.0 on line 36. Tested without this import and seems to work fine.